packages/scene-composer/src/utils/entityModelUtils/sceneUtils.ts

```diff
@@ -24,3 +30,37 @@ export const createSceneRootEntity = (): Promise<CreateEntityCommandOutput> | un
 
   return getGlobalSettings().twinMakerSceneMetadataModule!.createSceneEntity(input);
 };
+
+export const checkIfEntityAvailable = async (
+  entityId: string,
+  sceneMetadataModule: TwinMakerSceneMetadataModule,
+): Promise<boolean> => {
+  try {
+    await sceneMetadataModule.getSceneEntity({ entityId });
```

Comment:
Maybe you can return the response from the api instead of true?

Reply:
wouldn't API throw error when not succeeded? at least entity not found will throw